docs/user-guide.md

```diff
@@ -2136,6 +2179,10 @@ TASK [patch : Update OPatch in GRID_HOME]
 
 ... output truncated for brevity
 ```
+#### A note on patch metadata
+
+See [docs/quarterly_RU_update.md](docs/quarterly_RU_update.md) for details on
```

Comment:
Nit: can this link be relative?  The reason is that files here have different URL paths depending on how they're accessed:  google.github.io, github.com, or possibly others.

https://github.blog/2013-01-31-relative-links-in-markup-files/

Reply:
Changing the hyperlink to relative links by referencing it as `./` as follows:
> See [docs/quarterly_RU_update.md](./quarterly_RU_update.md) for details on

Will also change the file name from `docs/quarterly_RU_update.md` to `docs/patching-metadata.md` to match with other files names in:
```
12:33 $ ll docs/*
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan 134314 May  5 17:22 docs/bms-toolkit-architecture.png
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan   3194 May  5 17:22 docs/code-of-conduct.md
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan    327 May  5 17:22 docs/_config.yml
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan   1122 May  5 17:22 docs/contributing.md
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan    184 May  5 17:22 docs/index.html
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan   2178 Jun 30 14:23 docs/quarterly_RU_update.md
-rw-rw-r--. 1 jcnarasimhan jcnarasimhan  75103 Jun 30 14:23 docs/user-guide.md

```